docs/learn/lagandpacketloss.md

```diff
@@ -1,109 +1,56 @@
-
-:::note  
-While we can minimize the perceived impact of latency, we can't get rid of it - latency is just a fact of life that has to be taken into account.
-:::
-There are both network and non-network related components of latency.
-
-### Non-network Latency
+- **Input sampling delay**: The time it takes for an input device, such as a mouse or keyboard, to recognize that it's been activated and the time it takes for the game to detect that change.
+- **Render pipeline delay**: The time it takes for the GPU to render the outcome of inputs, which are often batched together to be processed in groups rather than individually.
+- **Vertical synchronization (VSync)**: A graphics technology that synchronizes the frame rate of the GPU with the refresh rate of the connected monitor. VSync can reduce undesirable visual behavior known as screen tearing, at the cost of increased latency incurred by the reduced frame rate.
+- **Display processing delay**: Display devices typically process incoming signals to some degree (such as deinterlacing and noise cancellation), which adds to latency.
+- **Pixel response time**: LCD screen pixels physically take time to change their brightness in response to inputs.
 
-Non-network latency is a serious issue and can eat up a large chunk of our ~200ms latency budget. This means that for online games input lag, and specifically the part that is largely under our control (render pipeline related lag) should be minimized.
+### Network latency
 
-- **Input sampling delay**: The time it takes for the input device to recognize that it has been activated and the time it takes for the game to detect that change
-- **Render pipeline delay**: GPUs don't perform draw commands immediately, instead they batch them to be performed later on
-- **Vsync**:  Prevents an artifact known as screen tearing by locking the GPU to the vertical blanking interval of the monitor
-- **Display processing delay**: Display devices typically process the incoming signal in some ways (such as deinterlacing and noise cancellation), which adds to latency
-- **Pixel response time**: LCD screen pixels physically take time to change their brightness
+Network latency refers to latency incurred by interactions over the network after local processing is complete.
 
-All in all a sum of non-network latency contributors can be called Input Lag - the time it takes user input to be rendered on screen (without any network communications being involved).
+Factors that contribute to network latency include:
 
-## Network latency
+- **Processing delay**: The time it takes for the network router to read packet headers and forward them to the appropriate location. Processing delay also includes any encryption, network address translation (NAT), or network mapping that the router might be doing. This delay is generally low, but can add up when packets are moving between multiple locations.
```

Comment:
```suggestion
- **Processing delay**: The time it takes for the network router to read packet headers and forward them to the appropriate location. Processing delay also includes any encryption, network address translation (NAT), or network mapping that the router might be doing. This delay is generally small, but can add up when packets move between multiple locations.
```

Reply:
I didn't like describing delay as "low". Feels wrong. It's a small amount of delay.... or a low delay value? Unsure, feel free to ignore if disagree.

Reply:
"short" also works.

Reply:
Nah, agree. Your suggestion is better.